### ISSUE TITLE
Ctrl-c crashing OMEdit on Windows 11

### ISSUE THREAD

**Issue by superlou** [OpenModelica/OpenModelica#11593]
### Description
After opening OMEdit, attempting to copy crashes OMEdit.

### Steps to reproduce
1. Open OMEdit. This was tried with version 1.21, 1.22, and today's 1.23 64-bit builds.
2. Press ctrl-c. I initially thought this occurred when I tried to copy and paste a block, but it appears that simply pressing ctrl-c causes the behavior.
3. The windows fades white to indicate the process is frozen. Eventually Windows prompts to kill the application.

### Expected behavior
The short-cut should copy the selected item.

### Version and OS
 - OpenModelica Version: OpenModelica v1.21.0 (64-bit), though also tried with 1.22 and 1.23.
 - Versions of used Modelica libraries if applicable: no additional libraries installed.
 - OS: Windows 11, 64-bit

### Additional context
I've used OpenModelica (1.21) for a while on my Windows 10 PC without this issue. I don't know if this is specifically a regression, or if it's something to do with the Windows version, which is why I tried rolling back to 1.21 after initially installing 1.22.

**Comment by adeas31** [comment]
That's weird. Anyways we are changing the way copy/paste is done as part of #11540 so hopefully new implementation fixes this issue as well. I will test this once the new functionality is in place.

**Comment by superlou** [comment]
I'll follow that issue. I'm somewhat convinced it has to be my computer, though I tried multiple versions to try and check when it would have changed. Will add more info if I can find what's specifically crashing. Thanks!

**Comment by adeas31** [comment]
I am not able to reproduce this issue. Can you please provide more information.

> Press ctrl-c. I initially thought this occurred when I tried to copy and paste a block, but it appears that simply pressing ctrl-c causes the behavior.

What are the selected items? Can you share the model?
What is the value of `Disable new instance-based graphical editing of models` in `Tools->Options->General`? Is it checked or unchecked?

**Comment by superlou** [comment]
`Disable new instance-based graphical editing of models` is unchecked. It actually doesn't seem to matter with any models. I can simply start OMEdit and press ctrl-c to hang the application.

Actually, might have something interesting: after I cleared recent models, pressing ctrl-c immediately after opening OMEdit doesn't case a hang. If I try to create a new model and use ctrl-c inside that new model, I get the hang. This happens still if I check `Disable new instance-based graphical editing of models.`

![image](https://github.com/OpenModelica/OpenModelica/assets/709695/967c78d4-5ae3-4960-a10c-61deeab5c17e)

**Comment by adeas31** [comment]
OK. That is very strange. Can it be that your system wide clipboard is somehow not accessible?

What happens if you right click any item in `Libraries Browser` and choose `Copy Path`? This should copy the path to the clipboard and then you can paste it on any text editor. Will this work for you?

**Comment by superlou** [comment]
Choosing `Copy Path` for an item in the `Libraries Browser` has the same hang. It does seem like any clipboard access causes it.

**Comment by adeas31** [comment]
Just wondering if you found any solution? Does copy/paste works on other applications?

**Comment by superlou** [comment]
No, unfortunately. I tried running OMEdit.exe from the command-line, and it doesn't seem to provide an debug information.

**Comment by adeas31** [comment]
What about the other applications? I am wondering if its Windows 11 general issue or is just your machine.

**Comment by superlou** [comment]
Ah, good point. I don't have issues with other applications, both in the Microsoft suite as well as other open source applications like Inkscape and Blender.

**Comment by adeas31** [comment]
OK. Can you try copy/paste OMNotebook? I am trying to narrow the issue scope. If it works in OMNotebook then something is wrong in OMEdit otherwise it could be a Qt issue.

**Comment by superlou** [comment]
Copy and paste in OMNotebook works fine.

**Comment by adeas31** [comment]
Geezzzz. Sorry, OMNotebook doesn't really use the system clipboard. Can you try one more thing,

Create a new model `M`, go to the text layer and presss ctrl+A and then ctrl+C. Does this copy the contents?

**Comment by superlou** [comment]
Copy and paste in the Text View with a brand new model works without crashing. If I go back to the Libraries Browser and right-click > copy the model, it hangs (just making sure my computer didn't fix it's crankiness).

**Comment by casella** [comment]
Good hunt 😄

**Comment by adeas31** [comment]
@superlou #11598 will show some messages if we fail to find the clipboard. You can test the nightly build tomorrow. Right click any item in `Libraries Browser` and choose `Copy Path`. Either the path will be copied to the clipboard without any error OR you should get some errors in `Messages Browser`.

**Comment by adeas31** [comment]
OK. Can you run it with gdb?

**Comment by superlou** [comment]
That's a good idea, though I've never used gdb on Windows. I'll look into how to do that. Still running into this on 1.22.3.
